src/exportImage.py

```diff
@@ -25,41 +29,46 @@ def export(self, offset, bbox, filename, url, taskid):
             return result
         except Exception as e:
             self.logger.error(f'Error occurred while exporting: {e}.')
+            doc = {
+                "body": {
+                    "taskId": taskid,
+                    "status": "failed",
+                    "datetime": str(datetime.now()),
+                    "filename": filename
+                }
+            }
+            self.update_db(doc)
             raise e
 
     def progress_callback(self, complete, message, unknown):
-        index = self.__config["es"]["index"]
-        hostip = self.__config["es"]["host_ip"]
-        port = self.__config["es"]["port"]
+        percent = floor(complete * 100)
+        doc = {
+            "body": {
+                "taskId": unknown["taskId"],
+                "status": "in-progress",
```

Comment:
can you create an enum for the statuses?
ask @alebinson  what fields he uses in order to translate it correctly in the client

Reply:
fixed in 79c5e7e